edx_sysadmin/models.py

```diff
@@ -1,23 +1,19 @@
 """
 Database models for edx_sysadmin.
 """
+from django.db import models
+from jsonfield.fields import JSONField
 
-import mongoengine
+from opaque_keys.edx.django.models import CourseKeyField
 
-from xmodule.modulestore.mongoengine_fields import CourseKeyField
 
+class CourseGitLog(models.Model):
+    """CourseGitLog to store git-logs of courses imported from github"""
 
-class CourseImportLog(mongoengine.Document):
-    """Mongoengine model for git log"""
-
-    course_id = CourseKeyField(max_length=128)
-    # NOTE: this location is not a Location object but a pathname
-    location = mongoengine.StringField(max_length=168)
-    import_log = mongoengine.StringField(max_length=20 * 65535)
-    git_log = mongoengine.StringField(max_length=65535)
-    repo_dir = mongoengine.StringField(max_length=128)
-    commit = mongoengine.StringField(max_length=40, null=True)
-    author = mongoengine.StringField(max_length=500, null=True)
-    date = mongoengine.DateTimeField()
-    created = mongoengine.DateTimeField()
-    meta = {"indexes": ["course_id", "created"], "allow_inheritance": False}
+    course_id = CourseKeyField(max_length=255, db_index=True)
+    import_log = JSONField(null=True, blank=True)
+    git_log = models.TextField(null=True, blank=True, max_length=65535)
+    repo_dir = models.CharField(max_length=255, db_index=True)
+    commit = models.CharField(max_length=40, null=True)
+    author = models.CharField(max_length=255, db_index=True)
```

Comment:
too many indexes? do we need index on `author` and `repo_dir` fields?

Reply:
Thanks for pointing this out, those were added to imitate the previous schema but right now we are fetching `git-logs` on the basis of `course_id` only so I have removed those indexing now.